### ISSUE TITLE
afew does not properly escape IDs with quotes

### ISSUE THREAD

**Issue by wad209** [afewmail/afew#116]
After I sync a fresh copy of my maildir (its kind of large, about 16k emails), I run notmuch new, which takes about a minute. Then I get an exception about 30s into afew --tag --new

```
afew --tag --new       
Traceback (most recent call last):
  File "/usr/bin/afew", line 9, in <module>
    load_entry_point('afew==0.0.0', 'console_scripts', 'afew')()
  File "/usr/lib/python3.5/site-packages/afew/commands.py", line 205, in main
    inner_main(options, database, query_string)
  File "/usr/lib/python3.5/site-packages/afew/main.py", line 39, in main
    filter_.commit(options.dry_run)
  File "/usr/lib/python3.5/site-packages/afew/filters/BaseFilter.py", line 113, in commit
    messages = notmuch.Query(db, 'id:"%s"' % message_id).search_messages()
  File "/usr/lib/python3.5/site-packages/notmuch/query.py", line 178, in search_messages
    raise NullPointerError
notmuch.errors.NullPointerError
```

I see that it is actually crashing in notmuch, but it seems to me the bug is with afew and not notmuch (although I could be wrong).

Here is my config

```
[SpamFilter]
#[ClassifyingFilter]
[KillThreadsFilter]
[ListMailsFilter]
[ArchiveSentMailsFilter]
[InboxFilter]
[MailMover]                 
folders = Inbox Spam
Inbox = 'tag:spam':Spam
Spam = 'NOT tag:spam':Inbox
```

I should note that [ClassifyingFilter] is commented out because it spews a ton of warnings. It segfaults either way, though.

**Comment by teythoon** [comment]
That is not a segmentation fault.  If it were it would definitively be a notmuch bug.  Can you please be more specific which version of notmuch are you using?

Nice to see you using afew with python3 :)

**Comment by wad209** [comment]
Oops, yeah you're right! I'm using notmuch 0.21. I'm on afew commit 3f1e5e9.

**Comment by wad209** [comment]
I should probably add that it seems to be a particular email or emails that is causing issues. If I don't sync my archived emails in gmail, everything works great.

**Comment by wad209** [comment]
Okay, well I found the issue. Afew searches for the id:"4b61bac74fbb7_266e..fdbfb70d81026"@asgrubyprod.cc.uga.edu.tmail

But the actual id:id:"""4af5ddf8a014e_7648..fdbe2437e171c""@puff.openstreetmap.org.tmail"

So it seems like an issue with how the escaping works.

**Comment by np** [comment]
I'm also experiencing that

**Comment by wad209** [comment]
I changed the name to properly describe the issue.

**Comment by maxigas** [comment]
Subscribing, have the same problem. I can test if needed. Thanks for the software!

**Comment by varac** [comment]
Same issue here.

**Comment by GuillaumeSeren** [comment]
Hey,
I would be interrested to know if the error still occur on the last release '1.1.0' ?

**Comment by chy-causer** [comment]
Can confirm I am running 1.1.0 with notmuch 0.25 and have just been bitten by this.

**Comment by flokli** [comment]
should be fixed by #203, can you check and reopen if it doesn't?
